packages/jest-puppeteer-axe/src/index.ts

```diff
@@ -54,31 +50,44 @@ function formatViolations( violations ) {
 		.join( '\n' );
 }
 
+interface AxeTestParams {
+	include?: string | string[];
+	exclude?: string | string[];
+	disabledRules?: string[];
+	options?: RunOptions;
+	config?: Spec;
+}
+
+interface MatcherContext {
+	utils: {
+		matcherHint: ( matcherName: string ) => string;
+		RECEIVED_COLOR: ( text: string ) => string;
+	};
+}
+
 /**
  * Defines async matcher to check whether a given Puppeteer's page instance passes Axe accessibility tests.
  *
  * @see https://www.deque.com/axe/
  * It is possible to pass optional Axe API options to perform customized check.
- *
  * @see https://github.com/dequelabs/axe-core-npm/tree/develop/packages/puppeteer
  *
- * @param {Page}          page                 Puppeteer's page instance.
- * @param {?Object}       params               Optional params that allow better control over Axe API.
- * @param {?string|Array} params.include       CSS selector(s) to add to the list of elements
- *                                             to include in analysis.
- * @param {?string|Array} params.exclude       CSS selector(s) to add to the list of elements
- *                                             to exclude from analysis.
- * @param {?Array}        params.disabledRules The list of Axe rules to skip from verification.
- * @param {?RunOptions}   params.options       A flexible way to configure how Axe run operates,
- *                                             see https://github.com/dequelabs/axe-core/blob/HEAD/doc/API.md#options-parameter.
- * @param {?Spec}         params.config        Axe configuration object,
- *                                             see https://github.com/dequelabs/axe-core/blob/HEAD/doc/API.md#api-name-axeconfigure.
- *
+ * @param {MatcherContext}  this                   Matcher context from Jest.
+ * @param {Page}            page                   Puppeteer's page instance.
+ * @param {Object}          [params]               Optional params that allow better control over Axe API.
+ * @param {string|string[]} [params.include]       CSS selector(s) to include in analysis.
+ * @param {string|string[]} [params.exclude]       CSS selector(s) to exclude from analysis.
+ * @param {string[]}        [params.disabledRules] List of Axe rules to skip from verification.
+ * @param {RunOptions}      [params.options]       Options to configure how Axe run operates,
+ *                                                 see https://github.com/dequelabs/axe-core/blob/HEAD/doc/API.md#options-parameter.
+ * @param {Spec}            [params.config]        Axe configuration object,
+ *                                                 see https://github.com/dequelabs/axe-core/blob/HEAD/doc/API.md#api-name-axeconfigure.
  * @return {Object} A matcher object with two keys `pass` and `message`.
```

Comment:
We can move these docs to the interface itself, like this

```ts
interface AxeTestParams {
	/**
	 * CSS selector(s) to include in analysis.
	 */
	include?: string | string[];
	// Other properties
}
```
Then we can simply use this here

```ts
/*
 * @param {import('jest').MatcherContext} this   - Matcher context from Jest.
 * @param {Page}                          page   - Puppeteer's page instance.
 * @param {AxeTestParams}                 params - Optional params that allow better control over Axe API.
 * @return A matcher object with two keys `pass` and `message`.
 */
```